sky/services/platform/filesystem.mojom

```diff
@@ -0,0 +1,38 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+[DartPackage="sky_services"]
+module filesystem;
+
+/// Returns commonly used locations on the filesystem
+interface Filesystem {
```

Comment:
This isn't really a file system.  It's a way to get some well-known paths.  Maybe `PathProvider` ?